### DISCUSSION TITLE
Wind Control?

### DISCUSSION THREAD

**Discussion by rice-with-beans** [niels747/2D-Weather-Sandbox#17] [Ideas]
Add a feature that can change wind speed. I know there already is a global slider to force wind (-1, 1) but I want it to be able to be forcibly moved quickly and you can control the size and intensity just like for example how you add/remove water vapor and brush intensity of 0.001 to 0.05 I believe. The global wind slider takes a long time and I want a tool that can force wind quickly and over a small or certain area and not globally in the entire simulation world.

**Comment by niels747** [comment]
Okay I get what you mean, could be useful. I will add it to the todo list.

**Reply by rice-with-beans** [reply]
Thanks for adding it! Works like a charm! NOW TO CAUSE OVER 40c IN COASTAL AREAS >:)
